OSDOCS-20096 added assembly/modules for spiffe helper (#112997)
openshift/openshift-docs: modules/zero-trust-manager-spiffe-helper-about.adoc

// Module included in the following assemblies:
//
// * security/zero_trust_workload_identity_manager/zero-trust-manager-spiffe-helper.adoc

:_mod-docs-content-type: CONCEPT
[id="zero-trust-manager-spiffe-helper-about_{context}"]
= SPIFFE Helper container image

[role="_abstract"]

> SPIFFE Helper is a utility that connects to the SPIFFE Workload API, fetches identity credentials, writes them to files on disk, and optionally notifies a workload when X.509 material is renewed. Use it with the {zero-trust-full} when your application cannot call the Workload API directly but can read TLS certificates from a shared volume.

Reviewer: ```suggestion
Use SPIFFE Helper with the {zero-trust-full} when your application cannot call the workload API directly but can read TLS certificates from a shared volume.

SPIFFE Helper is a utility that connects to the SPIFFE Workload API, fetches identity credentials, writes them to files on disk, and optionally notifies a workload when X.509 material is renewed. 
```
too long (exceeds the character limit), but also i think your job is the "use when" sentence anyway

Author: Done

> The {zero-trust-full} provides a supported SPIFFE Helper container image based on the upstream SPIFFE Helper. The configuration file format, command-line flags, and Workload API behavior are compatible with upstream.

Reviewer: ```suggestion
The {zero-trust-full} provides a supported SPIFFE Helper container image based on the upstream SPIFFE Helper. The configuration file format, command-line flags, and workload API behavior are compatible with upstream.
```
pretty sure workload is a generic term here?

Author: Done

SPIFFE Helper performs the following tasks:

. Connects to the SPIFFE Workload API, usually through the {spire-full} Agent socket exposed by the SPIFFE CSI driver.

> . Fetches X.509 SVIDs, JWT SVIDs, and/or JWT bundles from {spire-full}.

Reviewer: ```suggestion
. Fetches X.509 {svid-full}s, JSON Web Token (JWT) SVIDs, and JWT bundles from {spire-full}.
```
need to define terms here; no and/or (IBM)

Author: Done

. Writes credentials to files under a configured directory such as `cert_dir`.

> . Optionally notifies a workload when X.509 material is renewed in daemon mode.

Reviewer: ```suggestion
. Optionally notifies a workload when X.509 authentication credentials are renewed in daemon mode.
```
i think is what you mean?

Author: Done
